webpack.config.js

```diff
@@ -252,11 +255,8 @@ function* webpackConfig( env, argv ) {
 			} ),
 			new CreateFileWebpack( {
 				path: './dist',
-				fileName: 'config.json',
-				content: JSON.stringify( {
-					buildMode: flagMode,
-					features,
-				} ),
+				fileName: 'config.php',
+				content: `<?php return array( 'buildMode' => '${ flagMode }', 'features' => array( ${ formattedFeaturesToPHPArray } ), ); ?>`,
```

Comment:
Leaving a note here after talking through this with @aaemnnosttv: Let's create a "properly" formatted file here, with the license header - it might be good to rely on a template in JS, like we're doing already for the manifest file.

Reply:
Sounds like a plan :100: